### PR TITLE
Refactoring Notification doc page

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

* Refactor notification doc page to make it easier to understand
* Adding disclaimer concerning the fact that when using "is_match" you need to specify ".name" at the end of the tag to make it work.

### Motivation

It was missing

### Preview link

* https://docs-staging.datadoghq.com/gus/is_match_update/monitors/notifications/

### CODE REVIEW

File: content/monitors/notifications.md [hunk 1]
```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
...
+
+* See a complete list of contextual template variables available to your monitor by clicking the "Use message template variables" link or in the list of suggestions that appears when you type `{{` to begin a template variable name.
+
+* The tag template variables can also be used in the monitor titles (names), but the variables are only populated in the text of Datadog child events (not the parent, which displays an aggregation summary). 
+
+* Some tags identifying your triggering scope are automatically inserted into the title of your multi alert. If your scope is defined by a lot of tags, your alert title may end up being undesirably long.
+
+#### {{comparator}} template variables
+
+The `{{comparator}}` template variable's value is always a relational operator. It correlates to the relational value selected in the monitor's "Set alert conditions" section:
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: Should it be:

```
It corresponds to the relational value selected in the monitor's "Set alert conditions" section:
```

instead of

```
It correlates to the relational value selected in the monitor's "Set alert conditions" section:
```

File: content/monitors/notifications.md [hunk 2]
```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
...
+{{< img src="monitors/notifications/comparator_alert.png" alt="comparator_alert" responsive="true" popup="true" >}}
+
+For example, when an alert set to trigger when a value rises "above" 50, this syntax:
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: I would reword 

```
For example, when an alert set to trigger when a value rises "above" 50, this syntax:
```

as:

```
For example, when an alert is set to trigger when a value rises "above" 50, the following syntax:
```

File: content/monitors/notifications.md [hunk 3]
```diff
@@ -212,55 +213,59 @@ https://app.datadoghq.com/dash/integration/system_overview?tpl_var_scope=host:{{
...
-https://app.datadoghq.com/infrastructure/map?fillby=avg%3Acpuutilization&sizeby=avg%3Anometric&filter=cassandra 
-```
-The above link has more customizable options than your standard System Dashboard.  Here you have additional variables to define.  Most common variables passed into this URL are the following: 
-
-**Fillby, sizeby, filter**: Fill by can be defined by adding fillby:avg:<MetricName>.  Size by can be defined by adding sizeby:avg:<SecondMetricName>.  Filter can be used to specify a specific integration (i.e. Cassandra, mysql, apache, snmp, etc) by adding filter=<integration_name>
-{{< img src="monitors/notifications/hostmap_url.png" alt="hostmap_url" responsive="true" popup="true" >}}
+  ```
+  https://app.datadoghq.com/infrastructure/map?fillby=avg%3Acpuutilization&sizeby=avg%3Anometric&filter=cassandra 
+  ```
+  The above link has more customizable options than your standard System Dashboard.  Here you have additional variables to define.  Most common variables passed into this URL are the following **Fillby, sizeby, filter**:
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: Should the "Fillby" in `**Fillby, sizeby, filter**:` not be capitalized?

File: content/monitors/notifications.md [hunk 4]
```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
...
 
 These can also be seen in the "Use message template variables" help box in
 Step 3 of the monitor editor.
 
-### Tag variables for multi alerts
-When your monitor is a multi alert, instead of having a generic message (and finding the triggering tag scope in the alert query definition), a variable can be used in the message for explicitly identifying the triggering scope.
+#### `{{is_alert}}` & `{{is_warning}}`
 
-Here is an example of how you can use template variables for a multi alert:
+These variables use simple if-else logic to display a different message depending on whether the event is a trigger, warning, recovery, or no data notification with the following syntax:
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: I was a bit confused reading this sentence. Could it be reworded as follows?

```
These variables use simple `if-else` logic to display a different message depending on the event type.
```

File: content/monitors/notifications.md [hunk 5]
```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
...
-In order to save some space, you can Include triggering tags in notification title. This makes your notification title look like this:
+{{< img src="monitors/notifications/templateconditionalrecover.png" alt="template conditional recover" responsive="true" popup="true" >}}
 
-{{< img src="monitors/notifications/templatevar_short.png" alt="template var short" responsive="true" popup="true" >}}
+#### `{{is_recovery}}` or `{{is_alert_recovery}}` 
 
-Note that template variable content is escaped by default. If your variable
-contains JSON or code that you would NOT like to be escaped, then use triple braces instead of double braces (e.g. `{{{event.text}}}`).
+* `{{is_recovery}}` triggers and a monitor recovers indifferently either from a **WARNING** state or an **ALERT** state. 
+* `{{is_alert_recovery}}` triggers when a monitor recovers directly from an **ALERT** state to **OK** state.
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: Could you add the word "an" here like:

```md
* `{{is_alert_recovery}}` triggers when a monitor recovers directly from an **ALERT** state to an **OK** state. 
```

File: content/monitors/notifications.md [hunk 6]
```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
...
+{{< img src="monitors/notifications/templateconditionalrecover.png" alt="template conditional recover" responsive="true" popup="true" >}}
 
-{{< img src="monitors/notifications/templatevar_short.png" alt="template var short" responsive="true" popup="true" >}}
+#### `{{is_recovery}}` or `{{is_alert_recovery}}` 
 
-Note that template variable content is escaped by default. If your variable
-contains JSON or code that you would NOT like to be escaped, then use triple braces instead of double braces (e.g. `{{{event.text}}}`).
+* `{{is_recovery}}` triggers and a monitor recovers indifferently either from a **WARNING** state or an **ALERT** state. 
+* `{{is_alert_recovery}}` triggers when a monitor recovers directly from an **ALERT** state to **OK** state. 
+* `{{is_warning_recovery}}` triggers when a monitor recovers from a **WARNING** state to **OK** state
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: Same as the previous comment. Could you please add the word "an" here like:

```md
* `{{is_warning_recovery}}` triggers when a monitor recovers from a **WARNING** state to an **OK** state
```

File: content/monitors/notifications.md [hunk 7]
```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
...
+
+This means that if the monitor switches from ALERT to WARNING then OK state:
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: Could you please reword like this?

```
This means that if the monitor switches from an **ALERT** to a **WARNING** to an **OK** state:
```

Comment (AndrewMcBurney) [reply content/monitors/notifications.md]: I also bolded the state names because they were bolded in the previous section.

File: content/monitors/notifications.md [hunk 8]
```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
...
-  {{/is_match}}
+{{/is_match}}
 ```
 
 Here is an example of how you can give a different message depending on the triggering context:
 
 {{< img src="monitors/notifications/scope_match_editor.png" alt="scope match editor" responsive="true" popup="true">}}
 
-## Variable availability
+**Note**: To use `{{is_match}}` conditional to check if a `tag_variable` is **NOT** empty, append `.name` after your tag name, for instance:
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: Could you please add the word "the" in front of `{{is_match}}` like:

```md
**Note**: To use the `{{is_match}}` conditional to 
```

File: content/monitors/notifications.md [hunk 9]
```diff
@@ -212,55 +213,59 @@ https://app.datadoghq.com/dash/integration/system_overview?tpl_var_scope=host:{{
...
+  ```
+  https://app.datadoghq.com/infrastructure/map?fillby=avg%3Acpuutilization&sizeby=avg%3Anometric&filter=cassandra 
+  ```
+  The above link has more customizable options than your standard System Dashboard.  Here you have additional variables to define.  Most common variables passed into this URL are the following **Fillby, sizeby, filter**: 
+
+  * `fillby` is defined by adding `fillby:avg:<MetricName>`.  
+  * `sizeby` is defined by adding `sizeby:avg:<SecondMetricName>`.
+  * `filter` is used to specify a specific integration (i.e. Cassandra, mysql, apache, snmp, etc) by adding `filter=<integration_name>`
+  {{< img src="monitors/notifications/hostmap_url.png" alt="hostmap_url" responsive="true" popup="true" style="width:70%;">}}
+  The above colors fill the hexagons by `system.cpu.system`, it sizes the hexagons by `system.cpu.stolen` and add a filter to only include Cassandra hosts.
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: Could you please rephrase this as:

```
The above link fills the hexagons by `system.cpu.system`, sizes them by `system.cpu.stolen`, and adds a filter to only include Cassandra hosts.
```

File: content/monitors/notifications.md [hunk 10]
```diff
@@ -212,55 +213,59 @@ https://app.datadoghq.com/dash/integration/system_overview?tpl_var_scope=host:{{
...
+  https://app.datadoghq.com/dash/integration/<integration_name>?tpl_var_scope=host:{{host.name}}
+  ```
+  {{< img src="monitors/notifications/integration_url.png" alt="integration_url" responsive="true" popup="true" style="width:70%;">}}
 
-**Integration Dashboards**- If you are building Application or Integration specific Monitors, you can link to that specific Integration Dashboard as well as adding a scope for the host that triggered the monitor.  In the example below all that is necessary to populate is the *<integration_name> section for something like Cassandra, apache, SNMP, etc as well as providing the scope for the offending host:
+* **Manage Monitors Page** – To link to a Manage monitors page that displays all of the monitors for the host in question, define a link like below:  
+  ```
+  https://app.datadoghq.com/monitors/manage?q=scope:host:{{host.name}}
+  ```
+  The above links to all monitors for this host. You have other options available to further refine the link.
```
Comment (AndrewMcBurney) [inline content/monitors/notifications.md]: Should it read `to further refine the link` or `to further refine the query`?

File: content/monitors/notifications.md (hunk 1 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: 👍

File: content/monitors/notifications.md (hunk 2 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: 👍

File: content/monitors/notifications.md (hunk 3 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: 👍

File: content/monitors/notifications.md (hunk 4 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: 👍

File: content/monitors/notifications.md (hunk 5 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: 👍

File: content/monitors/notifications.md (hunk 6 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: 👍

File: content/monitors/notifications.md (hunk 7 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: well done 👌

File: content/monitors/notifications.md (hunk 8 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: 👍

File: content/monitors/notifications.md (hunk 9 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: 👍

File: content/monitors/notifications.md (hunk 10 above)
Comment (l0k0ms) [reply content/monitors/notifications.md]: further refine the link, but I changed the sentence
